packages/rancher-cis-benchmark/charts/package.yaml

```diff
@@ -0,0 +1,3 @@
+providesGVR: clusterscans.cis.cattle.io/v1
+generateCRDChart:
+  enabled: true
```

Comment:
This file should be above charts folder

Reply:
moved the file up, now the CI validate step fails. Is there anything missing @cbron

Reply:
CI failure is known separate issue being worked on

Reply:
CI issue is resolved now